Dockerfile

```diff
@@ -11,15 +11,13 @@ COPY requirements_test.txt .
 COPY ocrd_tesserocr ./ocrd_tesserocr
 COPY Makefile .
 RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    add-apt-repository ppa:alex-p/tesseract-ocr && \
-    apt-get update && \
-    apt-get -y install --no-install-recommends \
-    libtesseract-dev \
+    apt-get install -y --no-install-recommends software-properties-common && \
+    add-apt-repository -y ppa:alex-p/tesseract-ocr && \
+    apt-get install -y --no-install-recommends \
+    g++ libtesseract-dev make \
```

Comment:
`make` is provided by `ocrd/core`, no need to install it, you can uninstall it but it will make little difference I think.

Reply:
Fixed now, thank you.